ci/build_container/docker_push.sh

```diff
@@ -29,19 +29,19 @@ then
 
         for distro in ubuntu centos
         do
-            echo "Updating lyft/envoy-build-${distro} image"
+            echo "Updating envoyproxy/envoy-build-${distro} image"
             LINUX_DISTRO=$distro ./docker_build.sh
-            docker push lyft/envoy-build-${distro}:$CIRCLE_SHA1
-            docker tag lyft/envoy-build-${distro}:$CIRCLE_SHA1 lyft/envoy-build-${distro}:latest
-            docker push lyft/envoy-build-${distro}:latest
+            docker push envoyproxy/envoy-build-"${distro}":"$CIRCLE_SHA1"
```

Comment:
This PR I think is going to need to get split. There are no images yet in the new location. Can you split the PR to just change where all the images are pushed? Then once new base images are pushed we can change the pull locations?

Reply:
Unless I'm misunderstanding, I think we should be good with this PR followed by a second that updates `.circleci/config.yml` to use build images at `envoyproxy/envoy-build`. This PR updates where both the envoy and build image are published and CI still uses `lyft/envoy-build`: https://github.com/envoyproxy/envoy/blob/master/.circleci/config.yml#L8

Reply:
It looks like you swapped docker_build.sh and ci_steps.sh, etc. I think those are going to be broken until we push the first images to the new location?

Reply:
Additionally, I believe we need to revert ci/envoy_build_sha.sh since this greps .circleci/config.yaml which we won't update until the second PR